tests/floresta-cli/getblock.py

```diff
@@ -19,10 +19,10 @@ class GetBlockTest(FlorestaTestFramework):
     # pylint: disable=line-too-long
     serialized_data = "0100000000000000000000000000000000000000000000000000000000000000000000003ba3edfd7a7b12b27ac72c3e67768f617fc81bc3888a51323a9fb8aa4b1e5e4adae5494dffff7f20020000000101000000010000000000000000000000000000000000000000000000000000000000000000ffffffff4d04ffff001d0104455468652054696d65732030332f4a616e2f32303039204368616e63656c6c6f72206f6e206272696e6b206f66207365636f6e64206261696c6f757420666f722062616e6b73ffffffff0100f2052a01000000434104678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5fac00000000"
 
-    bits = "ffff7f20"
-    chainwork = "2"
+    bits = "207fffff"
+    chainwork = "0000000000000000000000000000000000000000000000000000000000000002"
```

Comment:
I think we could mine a couple of blocks (ideally more than one mining epoch) to check the chainwork calculation is right.